utils/create.c

```diff
@@ -203,13 +204,26 @@ int tr_main(int argc, char* argv[])
         tr_metaInfoBuilderSetPieceSize(b, piecesize_kib * KiB);
     }
 
+    printf(b->fileCount > 1 ? " %u files, %s\n" : " %u file, %s\n", b->fileCount,
```

Comment:
Since `fileCount` and `pieceCount` are declared as fixed-size `uint32_t`, not `unsigned int`, use the matching `PRIu32` format macro instead of `u` for them here (and for `current` below).